cni/pkg/nodeagent/podcgroupns_linux.go

```diff
@@ -26,3 +30,30 @@ func GetInode(fi fs.FileInfo) (uint64, error) {
 	}
 	return 0, fmt.Errorf("unable to get inode")
 }
+
+// Gets the `starttime` field from `/proc/<pid>/stat`.
+// Note that this value is ticks since boot, and is not wallclock time
+func GetStarttime(proc fs.FS, pidDir fs.DirEntry) (uint64, error) {
```

Comment:
I don't mind this much but using `procfs` may also make sense given we already depend on it

Reply:
Yeah I considered it, but it isn't mockable in the same way that the stuff we already have is.